website/docs/addon_houdini_artist.md

```diff
@@ -376,6 +379,42 @@ When the render is finished, all the caches would be located in your publish fol
 You can see them in the Loader.
 ![Houdini Farm Per Task](assets/houdini/artist/houdini_farm_cache_loader.png)
 
+### Publishing render to Deadline
+
+Rendering on the farm is supported for:
+- **Karma ROP**
+- **Mantra ROP**
+- **Redshift ROP**
+- **Arnold ROP**
+- **VRay ROP**
```

Comment:
```suggestion
The following renderers are supported for both local and farm renderings:
- **Karma**
- **Mantra**
- **Redshift **
- **Arnold**
- **VRay**
```

Reply:
Nice Idea. I think after applying your suggestion, we won't be able to merge this PR until the local render support PR merged. https://github.com/ynput/ayon-core/pull/328

Reply:
Let's mark it as resolved temporarily. 
We can make another PR to update Houdini docs when new features are added.